cmake/modules/CTestScript.cmake

```diff
@@ -0,0 +1,8 @@
+function(build_tests)
```

Comment:
sorry for the confusion. i think we can either put the body of this function into this file as it is. and let user to call it like
```console
ctest -S cmake/CTestScript.cmake ...
```
or just include this file at the top-level `CMakeLists.txt`. again, i still think it might be better to keep `include(CTest)` in this file.